src/layouts/Static.tsx

```diff
@@ -9,6 +9,7 @@ import TableOfContents from "@/components/TableOfContents"
 import Heading from "@/components/OldHeading"
 import Text from "@/components/OldText"
 import UpcomingEventsList from "@/components/UpcomingEventsList"
+import GitHubContributors from "@/components/FileContributorsGitHub"
```

Comment:
Can we call the import and filename the same? for consistency

Reply:
Yeah... this was the same structure from the Gatsby repo... can clean it up a bit though 👍